cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -221,6 +235,21 @@ public void deleteNamespace(final Id.Namespace namespaceId)
     LOG.info("All data for namespace '{}' deleted.", namespaceId);
   }
 
+  private void deleteAllSchedules(Id.Namespace namespaceId) throws NotFoundException, SchedulerException {
+    List<ApplicationSpecification> allAppSpec =
+      new ArrayList<ApplicationSpecification>(store.getAllApplications(namespaceId));
+
+    for (ApplicationSpecification appSpec : allAppSpec) {
+      for (ScheduleSpecification scheduleSpec : appSpec.getSchedules().values()) {
```

Comment:
do you think its better to have a method ```deleteAllSchedules(Id.Namespace namespaceId, ApplicaitonSpecification appSpec)``` , so you can delete schedules in an application, and you can use it here.